releasenotes/notes/deprecate-constants-5e6e2cfdb76662ba.yaml

```diff
@@ -0,0 +1,6 @@
+---
+deprecations:
+  - |
+    ``ddtrace.ext.sql.ROWS`` is deprecated. Use ``ddtrace.ext.db.ROWCOUNT`` instead.
+  - |
+    ``ddtrace.ext.cassandra.ROW_COUNT`` is deprecated. Use ``ddtrace.ext.db.ROWCOUNT`` instead.
```

Comment:
```suggestion
  - |
    dbapi: ``ddtrace.ext.sql.ROWS`` is deprecated. Use ``ddtrace.ext.db.ROWCOUNT`` instead.
  - |
    cassandra: ``ddtrace.ext.cassandra.ROW_COUNT`` is deprecated. Use ``ddtrace.ext.db.ROWCOUNT`` instead.

```

Reply:
should be all set, updated the release notes